app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,21 @@
+module SessionsHelper
+  def log_in(user)
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    if session[:user_id]
+      @current_user ||= User.find_by(id: session[:user_id])
+    end 
```

Comment:
this could also be written with one liner:

`@current_user ||= User.find_by(id: session[:user_id]) if session[:user_id]`